src/app/organizations/manage/policy-edit.component.html

```diff
@@ -36,7 +36,7 @@ <h2 class="modal-title" id="policiesEditTitle">{{'editPolicy' | i18n}} - {{name}
                     <div class="form-check">
                         <input class="form-check-input" type="checkbox" id="enabled" [(ngModel)]="enabled"
                             name="Enabled">
-                        <label class="form-check-label" for="enabled">{{'enabled' | i18n}}</label>
+                        <label class="form-check-label" for="enabled">{{getCheckboxDesc()}}</label>
```

Comment:
Only needed if the method in the component ends up getting changed too.

```suggestion
                        <label class="form-check-label" for="enabled">{{checkboxDesc}}</label>
```

Reply:
Updated!